docs/project-outline.md

```diff
@@ -0,0 +1,40 @@
+# Project Outline
+
+**Target Users**  
+- Independent travelers and tourists planning multi‑city trips in Ukraine (initial focus: Kyiv & Lviv)  
+- History, culture and food enthusiasts seeking personalized, image‑augmented journey plans  
+
+**Core Features**  
+- Conversational Interface: Text and image input for travel preferences
+- City Selection: Initial focus on Kyiv and Lviv with scalability for additional cities
+- Intelligent Itinerary Generation: AI-powered journey plans based on user interests, budget, and duration
+- Iterative Refinement: Context-aware adjustments to generated itineraries
+- Conversation History: Access to past interactions and generated itineraries
+- Multi-modal Input: Support for images (e.g., landmarks) to enhance context
+
+**Required Data / APIs**  
+- Static attractions data: MediaWiki, Google Places
+- Restaurant/Dining: Google Places
+- Transportation schedules: Google Maps API (train/bus routes)  
+- Events: Eventbrite, Facebook Events, local platforms 
```

Comment:
Consider that Facebook Events has limited access via API now. You might want to list “Tavily Search” or “Google Custom Search” as a backup.